R/utils.R

```diff
@@ -247,6 +247,17 @@ tikz_dict = function(path) {
 # but now also place to tweak default options
 fix_options = function(options) {
   options = as.strict_list(options)
+  # convert dashes in option names with dots
+  dashes = grep('-', names(options), value = TRUE)
+  options[gsub('-', '.', dashes)] = options[dashes]
+  options[dashes] = NULL
+
+  # normalize aliases
+  aliases = c(fig.dev = 'dev', fig.dpi = 'dpi')
```

Comment:
In quarto the alias is in fact `fig-format`

```suggestion
  aliases = c(fig.format = 'dev', fig.dpi = 'dpi')
```